### PR TITLE
bug(web): drag from arrow on desktop

### PR DESCRIPTION
**Pull request by tonypls**
## Issue
Unable to drag the map when clicking an exchange arrow

## Description
This PR updates the exchange arrows on desktop to have pointerEvents: none. This allows dragging of the map from desktop while still seeing the tooltip on hover. 

On mobile it's significantly more difficult to handle both tap to show the exchange tooltip and drag to move the map. I suggest this isn't worth the time as I believe it requires using the arrow to control the map or elaborate passing of events.

### CODE REVIEW

**Review by VIKTORVAV99 — APPROVED** [review]
LGTM! 🎉

**Comment by tonypls** [comment]
> LGTM! 🎉

It didn't actually work 🤦 so I went with the arrow controlling the map approach.

File: web/src/features/exchanges/ExchangeArrow.tsx [hunk 1]
```diff
@@ -30,6 +30,18 @@ function ExchangeArrow({
...
+    const cancelWheel = (event: Event) => event.preventDefault();
+    const exchangeLayer = document.querySelector('#exchange-layer');
+    if (!exchangeLayer) {
+      return;
+    }
+    exchangeLayer.addEventListener('wheel', cancelWheel, {
+      passive: true,
+    });
+    return () => exchangeLayer.removeEventListener('wheel', cancelWheel);
+  }, []);
```
Comment (VIKTORVAV99) [inline web/src/features/exchanges/ExchangeArrow.tsx:43]: Even more use effect hooks 🙈 

I'll test this after lunch!

Comment (tonypls) [reply web/src/features/exchanges/ExchangeArrow.tsx:43]: this one was already there, I had removed it tho! I also didn't have a mousewheel to test 😆

**Review by VIKTORVAV99 — APPROVED** [review]
LGTM! 🎉

Tested with a scroll wheel as well and it works!

**Pull request merged**